Lib/pdb.py

```diff
@@ -448,7 +448,8 @@ def _validate_file_mtime(self):
             return
         if (filename in self._file_mtime_table and
             mtime != self._file_mtime_table[filename]):
-            self.message(f"*** WARNING: file '{filename}' changed after pdb started")
+            self.message(f"*** WARNING: file '{filename}' was edited after pdb started, "
+                         "running stale code")
```

Comment:
Is the code necessarily loaded when PDB is started? Couldn't it be loaded later?

Reply:
I think we have an ambiguous word "code" here - do you mean source code? The source file is always up to date (to disk). The code object will not be updated.